dbt/models/proximity/docs.md

```diff
@@ -84,6 +84,14 @@ Distance from each PIN to the nearest golf course.
 **Primary Key**: `pin10`, `year`
 {% enddocs %}
 
+# dist_pin_to_grocery_store
+
+{% docs table_dist_pin_to_grocery_store %}
+Distance from each PIN to the nearest grocery_store.
```

Comment:
```suggestion
Distance from each PIN to the nearest grocery store.
```

Reply:
I would also add the OSM tags here used to define grocery store.

Reply:
> I would also add the OSM tags here used to define grocery store.

I assume this references dbt/models/proximity/columns.md.